recipe/meta.yaml

```
@@ -1,28 +1,43 @@
-{% set version = "2.1.2" %}
+{% set version = "3.0.0" %}
 
 package:
   name: pendulum
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/pendulum/pendulum-{{ version }}.tar.gz
-  sha256: b06a0ca1bfe41c990bbf0c029f0b6501a7f2ec4e38bfec730712015e8860f207
+  sha256: 5d034998dea404ec31fae27af6b22cff1708f830a1ed7353be4d1019bb9f584e
+  patches:
+    # https://github.com/conda-forge/pendulum-feedstock/commit/34b57a4e8a83577a1d7e25460444683aa3128071
+    - patches/0001-remove-tzdata-dep-for-pip-check.patch
 
 build:
-  number: 1
-  noarch: python
+  number: 0
+  skip: True  # [py<38]
   script:
-    - {{ PYTHON }} -m pip install . --no-deps -vv
+    - {{ PYTHON }} -m pip install . --no-deps -vv --no-build-isolation
+  missing_dso_whitelist:  # [s390x]
+    - "$RPATH/ld64.so.1"  # [s390x]
 
 requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+    - patch        # [not win]
+    - m2-patch    # [win]
   host:
-    - python >=3.6
+    - python
     - pip
-    - poetry-core
+    - maturin >=1.0,<2.0
+    - setuptools
+    - wheel
   run:
-    - python >=3.6
-    - python-dateutil >=2.6,<3.0
-    - pytzdata >=2020.1
+    - python
+    - python-dateutil >=2.6
+    - tzdata >=2020.1
```

```suggestion
    - python-tzdata >=2020.1
```
The python package tzdata is called python-tzdata on our channel, to not clash with contains timezone data itself.